app2.js

```diff
@@ -0,0 +1,35 @@
+const express = require("express")
+const cors = require("cors")
+const app = express()
+app.use(cors())
+const port = 3002;
+let add = (a,b)=>a+b;
+let subtract =(a,b)=>a-b;
+let multiply = (a,b)=>a*b;
+let divide = (a,b)=>a/b
+app.get("/:operator/:num1/:num2", (request,response)=>{
+let operator = request.params.operator;
+console.log(operator)
+let num1 = request.params.num1;
+let num2 = request.params.num2;
+let result;
+if(operator === "add"){
```

Comment:
I think the assignment wanted you to create different routes. But this is a really nice function.

Think about if adding an `else` block in